crates/storage/db/src/snapshot/masks.rs

```diff
@@ -0,0 +1,28 @@
+use super::{ReceiptMask, TransactionMask};
+use crate::{
+    add_snapshot_mask,
+    snapshot::mask::{ColumnSelectorOne, ColumnSelectorTwo, HeaderMask},
+    table::Table,
+    CanonicalHeaders, HeaderTD, Receipts, Transactions,
+};
+use reth_primitives::{BlockHash, Header};
+
+// HEADER MASKS
+
+add_snapshot_mask!(HeaderMask, Header, 0b001);
+add_snapshot_mask!(HeaderMask, BlockHash, 0b100);
+add_snapshot_mask!(HeaderMask, <HeaderTD as Table>::Value, 0b010);
```

Comment:
just to have them in order
```suggestion
add_snapshot_mask!(HeaderMask, <HeaderTD as Table>::Value, 0b010);
add_snapshot_mask!(HeaderMask, BlockHash, 0b100);
```